app/src/renderer/components/governance/PageProposal.vue

```diff
@@ -244,8 +252,14 @@ export default {
     }
   },
   methods: {
-    onVote() {
+    async onVote() {
       this.showModalVote = true
+
+      // The error is already handled with notifyError in votes.js
+      await this.$store.dispatch(`getProposalVotes`, this.proposalId)
```

Comment:
let's move this logic to `mounted()` instead, will need it afterwards for the `TabVotes` so better to make it available globally on the component, not just for the modal

Reply:
Currently the data should be available globally in the lastVote property.
I think it's more effective to keep the dispatching bounded with the action, if we move the call away from the action it won't be refreshed, let's say we move it to mounted:
- I vote Yes and the modal closes
- click again to vote without leaving the proposal page
- Any button is disabled because the component is already mounted and the lastVote has not been updated
- I can vote Yes again

Reply:
good argument, makes sense to me